### PR TITLE
Fixed your on_actions by merging them

### PR DESCRIPTION
**Pull request by SnowletTV**

### CODE REVIEW

File: common/on_action/ml_monthly_country.txt [hunk 1]
```diff
@@ -1,5 +0,0 @@
-﻿monthly_country_pulse = {
```
Comment (x3igh7) [inline common/on_action/ml_monthly_country.txt:1]: looks like you removed this and didn't add it back

File: common/on_action/00_on_invention.txt [hunk 2]
```diff
@@ -1,6 +0,0 @@
-﻿on_invention = {
```
Comment (x3igh7) [inline common/on_action/00_on_invention.txt:1]: this didnt get added back

File: common/on_action/ml_monthly_country.txt (hunk 1 above)
Comment (x3igh7) [inline common/on_action/ml_monthly_country.txt:1]: this is a monthly pulse not specific from code - need to undelete

File: common/on_action/on_game_initialized_field_of_mars.txt [hunk 3]
```diff
@@ -1,536 +0,0 @@
-﻿on_game_initialized = {
```
Comment (x3igh7) [inline common/on_action/on_game_initialized_field_of_mars.txt:1]: oof. this is a massive awful file. might need to diff this and see what's different from vanilla other the terrain stuff

File: common/on_action/00_on_invention.txt (hunk 2 above)
Comment (SnowletTV) [reply common/on_action/00_on_invention.txt:1]: Woops

File: common/on_action/ml_monthly_country.txt (hunk 1 above)
Comment (SnowletTV) [reply common/on_action/ml_monthly_country.txt:1]: I think I deleted a wrong file here yeah

File: common/on_action/on_game_initialized_field_of_mars.txt (hunk 3 above)
Comment (SnowletTV) [reply common/on_action/on_game_initialized_field_of_mars.txt:1]: This file was a pain to look at and i'm surprised it just worked tbh

**Pull request closed without merging**